Layout/UITableView+Layout.swift

```diff
@@ -510,13 +504,7 @@ extension UITableViewCell {
             let idExpression = LayoutExpression(expression: expression, type: RuntimeType(String.self), for: node)
             reuseIdentifier = try idExpression.evaluate() as? String
         }
-        let cell: UITableViewCell
-        if self == UITableViewCell.self {
-            cell = UITableViewCell(style: style, reuseIdentifier: reuseIdentifier)
-        } else {
-            cell = self.init() // Workaround for `self.init(style:reuseIdentifier:)` causing build failure
```

Comment:
This *does* actually let you create custom cell subclasses, it just doesn't let you use non-default style.